### PR TITLE
#93 [FIX] appendTo not working

### PR DESCRIPTION
**Pull request by unijad**
this should be fixing the appendTo issue

### CODE REVIEW

File: dist/date-range-picker.js [hunk 1]
```diff
@@ -1023,7 +1026,6 @@
     var dp = BaseMode(root, emit, opts);
 
     dp.close = noop;
-    dp.destroy = noop;
```
Comment (chrisdavies) [inline dist/date-range-picker.js]: I'm curious why this line is being removed? It seems unrelated to the `appendTo` fix, isn't it?

Comment (unijad) [reply dist/date-range-picker.js]: nop, suppose not to be deleted, only other change, I will resolve this one, and create new pull request, you may disqualify this one

Comment (unijad) [reply dist/date-range-picker.js]: it was removed in this commit 43313b820b989cc4ae8ca29e4de61b4f7947fe92 due to merging

**Comment by sporifolous** [comment]
This works great for me! Looking forward to getting a release with this fix.

**Comment by jmaicher** [comment]
Is there any chance of getting this merged?
